netman/adapters/switches/brocade.py

```diff
@@ -439,10 +439,12 @@ def parse_vlan(vlan_data):
     for line in vlan_data[1:]:
         if regex.match("^\srouter-interface ve (\d+)", line):
             current_vlan.vlan_interface_name = regex[0]
-        elif regex.match(" tagged (.*)", line):
-            for name in parse_if_ranges(regex[0]):
-                current_vlan.tagged_interfaces.append(name)
-
+        elif regex.match("^\s(tagged|no untagged)\s(.*)$", line):
+            type = regex[0]
+            for name in parse_if_ranges(regex[1]):
+                current_vlan.interfaces.append(name)
+                if type == "tagged":
+                    current_vlan.tagged_interfaces.append(name)
```

Comment:
The field interfaces is added within PR at f67d6591f4d7d094f1aeb87a3e2e8b62b99fe92b.

The tagged_interface was already on [VlanBrocade:53](https://github.com/internap/netman/blob/master/netman/adapters/switches/brocade.py#L513) and has no test, therefore I cannot tell why it's there and if I should remove it. We have to figure this out. I am not aware of *tagged vs untagged* and if we have to go trough both to change the vlan cleanup.